bedrock/settings/base.py

```diff
@@ -1466,6 +1493,12 @@ def facebook_tab_url_lazy():
 # Optimize.ly project code
 OPTIMIZELY_PROJECT_ID = None
 
+# Fx Accounts Relier Service Client ID
+# TODO: Update when available! Currently using creds for demo5 & URLs for FxA dev
```

Comment:
Yes, unfortunately. They'll go in `local.py` on each server anyway - will probably leave blank with comment in `base.py`.